extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.configuration.js

```diff
@@ -490,6 +490,13 @@ angular.module('PaperUI.controllers.configuration', [ 'PaperUI.constants' ]).con
             }
         });
     };
+
+    $scope.hasProperties = function(properties) {
```

Comment:
Yes, we can essentially create a utility service and create such functions there. It will also help refactor some code.